scripts/src/main/resources/scripts/environment-project.bat

```diff
@@ -79,10 +79,11 @@ rem %~1: variable name
 rem %~2: variable value (may contain ~ or ${var})
 :set_variable
 setlocal EnableDelayedExpansion
-set "value=%~2"
+set "value=%~2 %~3"
```

Comment:
Ah, if I get this correctly you were stumbling over situations like:
```
property="word1 word2"
```
And here `%~2` would be `"word1"` and `%~3` would be `word2"`, correct?

If that is the case we might need to do a `shift` command and then use the syntax to insert all parameters so it would also work for more words:
```
property="word1 word2 word3 word4 word5 word6 word7 word8 word9 word10 word11 word12"
```

WDYT?

Reply:
Hi @hohwille , You are right. For more then two words separated by spaces need to add more numbers of formal parameter like %~4 ,%~5..... etc.This is the fix only for a double quotes String like property="word1 word2" separated by space.

Reply:
The MSDos/CMD Syntax for all parameters is `%*`. This will include also the first one.
To prevent including the first one (the property name), use the `shift` command that shifts all arguments to the left. The first parameter is lost and becomse the second that becomes the third, etc.

Reply:
The `shift` command in MS CMD is entirely broken and does not affect `%*`. As always with CMD simple things are pain. I am still working on a solution and will redo this PR.

Reply:
I tried something like this:
https://stackoverflow.com/questions/935609/batch-parameters-everything-after-1
However, the shift command seems as a side effect to manipulate the arguments of the main program while we are in a sub-routine that has its own arguments for the variable assignment. CMD is nothing but ugly hacking...